test/studies/cholesky/jglewis/version2/elemental/fully_blocked/elemental_cholesky_fully_blocked.chpl

```diff
@@ -69,7 +69,7 @@ module elemental_cholesky_fully_blocked {
 
   use CyclicDist, Barriers;
 
-  public use blocked_elemental_schur_complement, 
+  use blocked_elemental_schur_complement, 
```

Comment:
switched to private use to avoid conflict between `module blocked_elemental_schur_complement` and `proc blocked_elemental_schur_complement`.